components/scanners/cdxgen/component.yaml

```diff
@@ -0,0 +1,30 @@
+name: cdxgen
+description: "Creates an SBOM then sends it to a remote Dependency Track"
+type: scanner
+parameters:
+  - name: backend_server_url
+    type: string
+    value: ""
+  - name: api_key
+    type: string
+    value: ""
+  - name: project_name
+    type: string
+    value: ""
+  - name: project_version
+    type: string
+    value: ""
+steps:
+  - name: run-cdxgen
+    image: ghcr.io/cyclonedx/cdxgen:latest
+
+    executable: "node"
```

Comment:
this will not work. you need an explicit path to the node binary

Reply:
it works, unless it changed between me testing and now.
but let me add a full path